install

```diff
@@ -162,8 +162,8 @@ group_chmods = %w( bin etc Frameworks include lib sbin share var
                    map { |d| File.join(HOMEBREW_PREFIX, d) }.select { |d| chmod?(d) }
 # zsh refuses to read from these directories if group writable
 zsh_dirs = %w( share/zsh share/zsh/site-functions )
-user_chmods = zsh_dirs.
-                  map { |d| File.join(HOMEBREW_PREFIX, d) }.select { |d| chmod?(d) }
+                  .map { |d| File.join(HOMEBREW_PREFIX, d) }
```

Comment:
Can you explain why `chmod?` was removed? I think it should still be part of this check otherwise you may try and set permissions on directories that don't exist or already have the right permissions.